### PR TITLE
Moving to nf-core

### PR DESCRIPTION
**Pull request by matq007**
Tasks:
- [x] Convert the old pipeline to new by running `nf-core`
- [x] Created tool for parsing resutls from tools
- [x] Created tool for generating section in MultiQC summarizing fusion findings
- [x] Updated Dockerfile
- [x] Using conda to install every possible tool
- [x] Moved test files to [test repository](https://github.com/nf-core/test-datasets/tree/rnafusion)

Tools:
- [x] STAR-Fusion
- [x] Fusioncatcher
- [x] FusionInspector

Tested but not implemented:
- [x] GFusion (requires some downgrades and hasn't been maintaned in a while)
- [x] suhrig/arriba (wasn't able to run the tool)

### CODE REVIEW

**Comment by ewels** [comment]
Changed the base to the `dev` branch. Should mean that the tests run..

**Comment by ewels** [comment]
Ah, I don't think that travis picks up on this info actually, so tests won't run until you add a new commit to this PR.

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Ignore everything
+**
+
+# Allowed files
```
Comment (ewels) [inline .dockerignore]: Interesting - not seen this before. What's it useful for?

Comment (matq007) [reply .dockerignore]: In our use case, we use container only for executing tasks. There is no need to include the code in each of the containers really from my point of view.

**Pull request merged**